python/ray/utils.py

```diff
@@ -496,6 +497,141 @@ def is_main_thread():
     return threading.current_thread().getName() == "MainThread"
 
 
+def set_psigdeath(child_proc, _globals=[None]):
```

Comment:
Is the _globals argument passed in anywhere? If not can we remove it?

Reply:
No, it has to have global lifetime for the semantics to be consistent across platforms. Windows ties the child process lifetime to job objects, and Linux ties child process lifetimes to parent process lifetimes.